installer/install.sh

```diff
@@ -139,7 +139,7 @@ check_machine_image()
 update_syseeprom()
 {
     # Set platform name and ONIE version to sys_eeprom.
-    if [ -x /usr/bin/onie-syseeprom ] ; then
+    if [ -x /usr/bin/onie-syseeprom -a -f /dev/mtd-board_eeprom ] ; then
```

Comment:
Why is this needed?

As written here, this will break all of x86, none of which have /dev/mtd-board_eeprom.  All of x86 uses i2c to access the onie syseeprom.

If you want to skip update_syseeprom() for a particular platform there are ways to do that.

Reply:
nxp_ls1088ardb update was getting:

ERROR: Problems accessing sys_eeprom
Can't open /dev/mtd-board_eeprom: No such file or directory
Notice:  Invalid TLV header found.  Using default contents.
Notice:  Invalid TLV checksum found.  Using default contents.
Failure: Unable to install image: tftp://192.168.2.1/onie-updater-arm64-nxp_ls1088ardb-r0

But it looks like the correct fix is to remove the busybox onie-syseeprom config until correct i2c eeprom access is enabled.  (./machine/nxp/nxp_ls1088ardb/busybox/conf/config)